lms/static/sass/shared/_header.scss

```diff
@@ -357,7 +357,7 @@ header.global {
 // ====================
 
 // CASE: marketing/course discovery
-header.global-new {
+.header-global {
```

Comment:
I agree @andy-armstrong, it isn't ideal to have to add '-new' but it is the cleanest way I can think of keeping the legacy styles working. And as we move forward hopefully we can either clean this up or remove it completely and move to a single ```.header-global```. I know there is a future redesign of the header so I can push for cleanup in that story when we groom it.